web/src/resources/osk/kmwosk.css

```
@@ -196,7 +196,7 @@
 .tablet .kmw-key {border-radius:8px;overflow:visible;border:solid 2px #999999;box-shadow: 0px -1px 1px 0px rgba(0,0,0,1) inset;}
 
 .tablet .kmw-key-label{position:absolute;left:7%;top:2%;font:0.5em Arial;color:#aaa;background-color:transparent;z-index:10000;}
-.tablet .kmw-key-text{position:relative;left:0;top:0;-webkit-user-select:none;}
+.tablet .kmw-key-text{-webkit-user-select:none;}
```

@mcdurdin I just noticed that tablet key hints were being adversely affected when these lines were left in, but now they seem fine.  I didn't notice any other cross-effects from the change either, despite now non-specific this change is.    So... I just added this in.

I don't think a CSS `unset` is possible; we need the non-`.kmw-key-text` properties on the same element as specified for the `.kmw-key-popup-icon` class (visible below due to accidental extra line).  Both classes are applied to the key-hint element for most cases - that ensures key hints get the right font.

Anything I should look out for that I might not have checked on first view?

---

I don't know, the CSS was subtle in the past to deal with many inconsistencies cross-browser. Most of those inconsistencies are in the dim dark past of webland and probably no longer relevant. If it works correctly on all our supported browsers (i.e. min and max ver of each supported browser) then I think we'll be fine.

Note: we should be reformatting this file as we update it, e.g.:

```suggestion
.tablet .kmw-key-text {
  -webkit-user-select:none;
}
```

Also be good to be documenting the reasons for various rules, wrt caniuse.com. e.g. I think `-webkit-user-select` is very old now.